package.json

```diff
@@ -43,6 +43,14 @@
     "hubot": "./bin/hubot"
   },
   "scripts": {
-    "test": "./script/test"
+    "clean": "rm -rf lib/",
+    "preinstall": "script/dotenv",
```

Comment:
I just tested and it looks like it doesn't. I'm guessing you're right that it forks. 😞 

It looks like `.env` was only loaded from `script/bootstrap` before. Do you now what this was used for?

#1154 was proposing adding `.env` support for all of Hubot, which seems like something we might want to consider.

I can work on a fix for this tomorrow. Let me know if you have any idea what bootstrap was using .env for.